dojo/tools/cred_scan/parser.py

```diff
@@ -25,38 +30,30 @@ def get_findings(self, filename, test):
 
         dupes = dict()
         for row in reader:
-            # Severity is not provided in this scanner, so marking all as info
-            severity = 'Info'
-
             # Create the description
             description = row.get('Description', 'Description not provided')
-            is_suppressed = row.get('IsSuppressed', None)
-            supress_justification = row.get('SuppressJustification', None)
-            matching_score = row.get('MatchingScore', None)
-            date = row.get('TimeofDiscovery', None)
-
             # Add contextual details to the description
-            if is_suppressed:
-                description += '\n Is Supressed: ' + str(is_suppressed)
-            if supress_justification:
-                description += '\n Supress Justifcation: ' + str(supress_justification)
-            if matching_score:
-                description += '\n Matching Score: ' + str(matching_score)
+            if 'IsSuppressed' in row:
+                description += '\n Is Supressed: ' + str(row['IsSuppressed'])
+            if 'SuppressJustification' in row:
+                description += '\n Supress Justifcation: ' + str(row['SuppressJustification'])
+            if 'MatchingScore' in row:
+                description += '\n Matching Score: ' + str(row['MatchingScore'])
 
             finding = Finding(
-                    title=row.get('Searcher'),
+                    title=row['Searcher'],
                     description=description,
-                    severity=severity,
+                    severity='Info',
                     nb_occurences=1,
-                    file_path=row.get('Source'),
-                    line=row.get('Line'),
+                    file_path=row['Source'],
+                    line=row['Line'],
             )
             # Update the finding date if it specified
-            if date:
-                finding.date = datetime.strptime(date.replace('Z', ''), '%Y-%m-%dT%H:%M:%S.%f')
+            if 'TimeofDiscovery' in row:
+                finding.date = parser.parse(row['TimeofDiscovery'].replace('Z', ''))
```

Comment:
```suggestion
                finding.date = parser.parse(row['TimeofDiscovery'])
```
I think you can keep the timezone part. `python-dateutil` package is able to manage it I think.